api/src/main/resources/workbench.yaml

```diff
@@ -84,6 +84,140 @@ paths:
           schema:
             $ref: "#/definitions/Profile"
 
+  # Notebook clusters ####################################################################
+
+  /api/v1/clusters:
+    get:
+      summary: List all clusters
+      description: List all clusters, optionally filtering on a set of labels
+      operationId: listClusters
+      tags:
+        - cluster
+      parameters:
+        - in: query
+          name: _labels
+          description: |
+            Optional label key-value pairs to filter results by. Example: key1=val1,key2=val2.
+
+            Note: this string format is a workaround because Swagger doesn't support free-form
+            query string parameters. The recommended way to use this endpoint is to specify the
+            labels as top-level query string parameters. For instance: GET /api/clusters?key1=val1&key2=val2.
+          required: false
+          type: string
+      responses:
+        "200":
+          description: List of clusters
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cluster'
+        "400":
+          description: Bad Request
+          schema:
+            $ref: '#/definitions/ErrorReport'
```

Comment:
We aren't ever actually populating an ErrorReport in our responses. (Which I think is fine, but we should remove it and its children from here.)

Reply:
That's fair. I do want to do a better job of handling errors, but was planning on doing that in a follow up PR. (The errors we get are somewhat obtuse because of the proxying)